Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerOsInformation.ps1

```diff
@@ -101,70 +101,85 @@ Function Invoke-AnalyzerOsInformation {
 
     foreach ($pageFile in $osInformation.PageFile) {
 
-        $maxPageSize = $pageFile.MaximumSize
-        Write-Verbose "Max PageFile Size: $maxPageSize"
-        $testingValue = [PSCustomObject]@{
+        $pageFileDisplayLayout = "'{0}' Size: '{1}MB'"
+        $pageFileAdditionalDisplayValue = $null
+
+        Write-Verbose "Max PageFile Size: $($pageFile.MaximumSize)"
+        $pageFileObj = [PSCustomObject]@{
+            Name                = $pageFile.Name
             TotalPhysicalMemory = $totalPhysicalMemory
-            MaxPageSize         = $maxPageSize
-            MultiPageFile       = $osInformation.PageFile.Count -gt 1
+            MaxPageSize         = $pageFile.MaximumSize
+            MultiPageFile       = (($osInformation.PageFile).Count -gt 1)
             RecommendedPageFile = 0
         }
 
-        if ($maxPageSize -eq 0) {
+        if ($pageFileObj.MaxPageSize -eq 0) {
             Write-Verbose "Unconfigured PageFile detected"
-            $displayValue = "Error: System is set to automatically manage the PageFile size."
+            $displayValue = ($pageFileDisplayLayout -f "Error -- System-managed", $pageFileObj.MaxPageSize)
             $displayWriteType = "Red"
         }
```

Comment:
do an `else` statement here and set the `$displayValue` to `"'$($pageFileObj.Name)' Size: '$($pageFileObj.MaxPageSize)'MB"

Then remove all the ones below.